[2023-08-13T08:00:15.173+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-08-13T08:00:15.239+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: userCase2.s3_file_check scheduled__2023-08-13T07:00:00+00:00 [queued]>
[2023-08-13T08:00:15.255+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: userCase2.s3_file_check scheduled__2023-08-13T07:00:00+00:00 [queued]>
[2023-08-13T08:00:15.255+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 3
[2023-08-13T08:00:15.271+0000] {taskinstance.py:1327} INFO - Executing <Task(S3KeySensor): s3_file_check> on 2023-08-13 07:00:00+00:00
[2023-08-13T08:00:15.278+0000] {standard_task_runner.py:57} INFO - Started process 9257 to run task
[2023-08-13T08:00:15.289+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'userCase2', 's3_file_check', 'scheduled__2023-08-13T07:00:00+00:00', '--job-id', '572', '--raw', '--subdir', 'DAGS_FOLDER/userCase.py', '--cfg-path', '/tmp/tmpyle96eux']
[2023-08-13T08:00:15.292+0000] {standard_task_runner.py:85} INFO - Job 572: Subtask s3_file_check
[2023-08-13T08:00:15.331+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-13T08:00:15.375+0000] {task_command.py:410} INFO - Running <TaskInstance: userCase2.s3_file_check scheduled__2023-08-13T07:00:00+00:00 [running]> on host 484e6be077e8
[2023-08-13T08:00:15.386+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-08-13T08:00:15.466+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='userCase2' AIRFLOW_CTX_TASK_ID='s3_file_check' AIRFLOW_CTX_EXECUTION_DATE='2023-08-13T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-13T07:00:00+00:00'
[2023-08-13T08:00:15.467+0000] {s3.py:103} INFO - Poking for key : s3://myawsbucket19080/raw/movies.csv
[2023-08-13T08:00:15.480+0000] {base.py:73} INFO - Using connection ID 'aws_connection' for task execution.
[2023-08-13T08:00:15.480+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_connection', conn_type='aws') credentials retrieved from login and password.
[2023-08-13T08:00:16.055+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2023-08-13T08:00:16.064+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=userCase2, task_id=s3_file_check, execution_date=20230813T070000, start_date=20230813T080015, end_date=20230813T080016
[2023-08-13T08:00:16.137+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-13T08:00:16.181+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
